controllers/modelmesh/runtime_test.go

```diff
@@ -73,30 +95,54 @@ func TestOverlayMockRuntime(t *testing.T) {
 				Spec: corev1.PodSpec{
 					Containers: []corev1.Container{
 						{
-							Name: "mm",
+							Name: adapterType,
```

Comment:
nit: Not a big deal, but shouldn't this be maintained as "mm" to coincide with the actual initial container name?

Reply:
Ah, I think I changed this in the wrong spot... With the addition of BuiltInAdapter, there should be a container with the same name as the `ServerType` (though this is checked well before the real code path would reach this point), so the runtime container is the thing that should have a matching name.

Nice catch.